contrib/awssqs/samples/README.md

```diff
@@ -0,0 +1,117 @@
+# AWS SQS - Source
+
+These samples demonstrate how to configure an AWS SQS source, consuming
+events from an AWS SQS queue.
+
+## Deployment Steps
+
+The steps below assume that you have cloned the
+[eventing-sources](https://github.com/knative/eventing-sources) repo and
+run the commands from contrib/awssqs.
+
+### Prerequisites
+
+1.  Create an [AWS SQS queue](https://aws.amazon.com/sqs/).
+
+1.  Setup
+    [Knative Eventing](https://github.com/knative/docs/tree/master/eventing).
+
+1.  The [in-memory `ClusterChannelProvisioner`](https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel)
+    should be installed in your cluster. At the time of writing (release
+    v0.4.0) it is part of the default instructions so you will probably
+    have it there already.
+
+### Create a channel and subscriber
+
+```shell
+ko apply -f samples/display-resources.yaml
+```
+
+The sample provided will configure an in-memory channel (named
+`awssqs-test` and a subscription to a consumer named `event-display`)
+that will simply display incoming events to stdout.
+
+At this point we have a channel and a subscriber ready to receive
+events, we will now create an awssqs source resource that starts
+pulling events from a SQS queue.
+
+### Set up credentials
+
+Acquire [AWS Credentials](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html)
+for the same account. Your credentials file should look like this:
+
+     [default]
+     aws_access_key_id = ...
+     aws_secret_access_key = ...
+
+Then create a secret for the downloaded key:
+
+ ```shell
+ kubectl -n knative-sources create secret generic awssqs-source-credentials --from-file=credentials=PATH_TO_CREDENTIALS_FILE
+ ```
+
+### Deployment
+
+Deploy the `AwsSqsSource` controller as part of eventing-source's controller.
+
+   ```shell
+   ko -n default apply -f config/
+   ```
+
+Note that if the `Source` Service Account secret is in a non-default
+location, you will need to update the YAML first.
+
+Replace the place holders in `samples/awssqs-source.yaml`.
+
+- `QUEUE_URL` should be replaced with your AWS SQS queue URL.
+
+  ```shell
+  export QUEUE_URL=https://sqs-eu-west-1.amazonaws.com/1234234234/my-queue
+  sed -i "s|QUEUE_URL|$QUEUE_URL|" awssqs-source.yaml
+  ```
+
+- `QUEUE_NAME` will be used to name the event source, you can choose any
+  value that makes sense to you, although a good choice is the last segment
+  of the URL.
+
+  ```shell
+  export QUEUE_NAME="my-queue"
+  sed -i "s|QUEUE_NAME|$QUEUE_NAME|" awssqs-source.yaml
+  ```
+
+- `awsCredsSecret` should be replaced with the name of the k8s secret that
+  contains the AWS credentials. Change this only if you deployed an altered
+  `config/default-awssqs.yaml` source config.
+
+Now deploy `awssqs-source.yaml`.
+
+```shell
+ko apply -f samples/awssqs-source.yaml
+```
+
+You can use [kail](https://github.com/boz/kail/) to tail the logs of the
+subscriber.
```

Comment:
Other sources use `kubectl log` to monitor the logs of event-display other than asking people to install a new tool `kail`. I will suggest to use `kubectl log` here for consistency. Of course, you can also add guidances about `kail` together with `kubectl log`.

Reply:
Kail is used in all the eventing and eventing-sources documentation, and the previous version of awssqs (this doc is mostly being moved).  Examples are [gcp pubsub](https://github.com/knative/eventing-sources/blob/master/contrib/gcppubsub/samples/README.md) L153 or [Camel](https://github.com/knative/eventing-sources/blob/master/contrib/camel/samples/README.md) at L73, L115.

I agree though that the less tools we expect the better, I can do a round and update all these docs, so we remain consistent?

Reply:
+1 for removing kail, we've been removing it as we go forward, but things linger. Perhaps we can just create an issue to track removing it from all the examples and do it in a followup.